### PR TITLE
STAC-21827: Add docs for external secrets

### PR DESCRIPTION
**Pull request by craffit**

### CODE REVIEW

File: setup/security/authentication/file.md [hunk 1]
```diff
@@ -84,9 +84,44 @@ Follow the steps below to configure users and apply changes:
...
+kind: Secret
+metadata:
+   name: "<custom-secret-name>"
+type: Opaque
+data:
+  file_<user1_with_non_alphanum_replaced_by_underscore>_password: <base64 of bcrypt of password>
+  file_<user2_with_non_alphanum_replaced_by_underscore>_password: <base64 of bcrypt of password> 
+```
+
+For every user in the logins section, a record should be added to the secret, where non alphanumeric passwords get replaced by an underscore. For example:
```
Comment (rb3ckers) [inline setup/security/authentication/file.md]: I think this is a bit to succinctly described. The non-alphanumeric bit doesn't refer to the password but to the username of the user for whom the password is set.

File: setup/security/authentication/service_tokens.md [hunk 2]
```diff
@@ -100,6 +100,21 @@ Follow the steps below to configure SUSE Observability to create a bootstrap ser
 * The authentication configuration is stored as a Kubernetes secret.
 {% endhint %}
 
+#### Setup the bootstrap service token though an external secret
```
Comment (rb3ckers) [inline setup/security/authentication/service_tokens.md]: Typo: though -> throug (or maybe "from" instead?)

File: k8s-suse-rancher-prime.md [hunk 3]
```diff
@@ -151,7 +151,7 @@ The `sizing.profile` should be one of trial, 10-nonha, 20-nonha, 50-nonha, 100-n
 This command will generate a `$VALUES_DIR/suse-observability-values/templates/baseConfig_values.yaml` and a `$VALUES_DIR/suse-observability-values/templates/sizing_values.yaml` file which contains the necessary configuration for installing the SUSE Observability Helm Chart.
 
 {% hint style="info" %}
-The SUSE Observability administrator passwords will be autogenerated by the above command and are output as comments in the generated `basicConfig.yaml` file. The actual values contain the `bcrypt` hashes of those passwords so that they're securely stored in the Helm release in the cluster.
+The SUSE Observability administrator password will be autogenerated by the above command and are output as comments in the generated `basicConfig.yaml` file. For more info, see [single password](/setup/security/authentication/single_password.md). The actual values contain the `bcrypt` hashes of those passwords so that they're securely stored in the Helm release in the cluster.
```
Comment (rb3ckers) [inline k8s-suse-rancher-prime.md]: Maybe add a second reference to the main "other authentication options" page to encourage users to not rely on this simple non-production setup.

File: setup/security/authentication/file.md (hunk 1 above)
Comment (craffit) [reply setup/security/authentication/file.md]: replaced this after your helm comment, we do not need any of this now after limiting the characters in the username

File: setup/security/authentication/service_tokens.md (hunk 2 above)
Comment (craffit) [reply setup/security/authentication/service_tokens.md]: done

File: k8s-suse-rancher-prime.md (hunk 3 above)
Comment (craffit) [reply k8s-suse-rancher-prime.md]: done

File: k8s-suse-rancher-prime.md [hunk 4]
```diff
@@ -151,7 +151,12 @@ The `sizing.profile` should be one of trial, 10-nonha, 20-nonha, 50-nonha, 100-n
...
 
 {% hint style="info" %}
-The SUSE Observability administrator password will be autogenerated by the above command and are output as comments in the generated `basicConfig.yaml` file. For more info, see [single password](/setup/security/authentication/single_password.md). The actual values contain the `bcrypt` hashes of those passwords so that they're securely stored in the Helm release in the cluster.
+The SUSE Observability administrator password will be autogenerated by the above command and are output as comments in the generated `basicConfig.yaml` file. For more info, see [single password](/setup/security/authentication/single_password.md).
+The actual values contain the `bcrypt` hashes of those passwords so that they're securely stored in the Helm release in the cluster.
+{% endhint %}
+
+{% hint style="warn" %}
+Using a single default password is gerat to get started with SUSE Observability, but for a production setup [more secure authentication options](/setup/security/authentication/authentication_options.md) are available.
```
Comment (rb3ckers) [inline k8s-suse-rancher-prime.md]: A typo in `gerat` -> `great`

**Review by rb3ckers — APPROVED** [review]

**Pull request merged**